odoo/models.py

```diff
@@ -1812,8 +1812,7 @@ def search(self, domain, offset=0, limit=None, order=None, count=False):
         :returns: at most ``limit`` records matching the search criteria
         :raise AccessError: * if user tries to bypass access rules for read on the requested object.
         """
-        res = self._search(domain, offset=offset, limit=limit, order=order, count=count)
-        return res if count else self.browse(res)
+        return 1 if count else self.browse([1])
```

Comment:
to make it even faster :

```suggestion
        return 1 if count else [1]
```

Reply:
Nope, it should return a recordset.  This avoids `browse` and use the _faster_
```suggestion
        return 1 if count else self.__class__(self.env, (1,), (1,))
```